.circleci/config.yml

```diff
@@ -816,27 +821,32 @@ jobs:
           platform: "x86_64"
           cudnn-version: << pipeline.parameters.cudnn-version >>
           trt-version-short:  << pipeline.parameters.trt-version-short >>
-          bazel-version: << pipeline.parameters.bazel-version >>
-          bazel-platform: "x86_64"
+          bazel-platform: "amd64"
       - install-torch-from-index:
           torch-build:  << parameters.torch-build >>
           torchvision-build: << parameters.torchvision-build >>
           torch-build-index: << parameters.torch-build-index >>
       - when:
           condition: << parameters.cxx11-abi >>
           steps:
-            - build-py-cxx11-abi
+            - build-py-cxx11-abi:
+                cuda-version: << parameters.cuda-version >>
+                torch-build-index: << parameters.torch-build-index >>
       - unless:
           condition: << parameters.cxx11-abi >>
           steps:
             - when:
                 condition: << parameters.legacy >>
                 steps:
-                  - build-py-legacy
+                  - build-py-legacy:
+                      cuda-version: << parameters.cuda-version >>
+                      torch-build-index: << parameters.torch-build-index >>
```

Comment:
The `legacy` CUDA version and Torch build index will likely differ from the stable version. Consider adding separate parameters for the legacy build (`parameters.cuda-version-legacy`)

Reply:
this would be the case for cudnn too right? I think it should be a separate PR